IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -27,11 +27,13 @@
 #import "MSIDDeviceId.h"
 #import <pthread.h>
 
+static long s_maxQueueSize = 1000;
+
 @interface MSIDLogger()
 
 @property (nonatomic) dispatch_queue_t loggerQueue;
+@property (nonatomic) dispatch_semaphore_t queueSemaphore;
 @property (nonatomic, copy) MSIDLogCallback callback;
-;
```

Comment:
Thanks for removing this, I saw it before but didn't want to submit a PR just to remove this :)